File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        
        for(String ch : babbling){
            ch = ch.replace("aya"," ");
            ch = ch.replace("ye"," ");
            ch = ch.replace("woo"," ");
            ch = ch.replace("ma"," ");
            ch = ch.trim();
            
            if(ch.equals("")){
                answer += 1;
                
            }
            
        }
        
        return answer;
    }
}

File: C#/프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.cs
using System;

public class Solution {
    public int solution(string[] babbling) {
            int answer = 0;

            for(int i = 0; i < babbling.Length; i++)
            {
                babbling[i] = babbling[i].Replace("aya", " ");
                babbling[i] = babbling[i].Replace("ye", " ");
                babbling[i] = babbling[i].Replace("woo", " ");
                babbling[i] = babbling[i].Replace("ma", " ");

                if (babbling[i].Trim() == "") answer++;
            }

            return answer;
    }
}